lambda/rest-endpoints/src/helpers.ts

```diff
@@ -0,0 +1,12 @@
+export function getBodyContentAsJson(body:unknown): string {
+  if(body instanceof Buffer) {
+    return body.toString('utf-8')
+  } else if(typeof body==='string') {
+    JSON.parse(body);
```

Comment:
`JSON.parse()` does not have a side effect I know of, is this line necessary?

Reply:
It will throw an exception if the string is not valid JSON, that's the reason

Reply:
Oh, that's neat. Should probably do it for a Buffer, too, perhaps?